codesystems/CodeSystem-England-ERSErrorCodes.xml

```diff
@@ -0,0 +1,28 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-ERSErrorCodes" />
+    <url value="https://fhir.nhs.uk/CodeSystem/England-ERSErrorCodes" />
+    <version value="1.0.0" />
+    <name value="EnglandERSErrorCodes" />
+    <title value="England ERS Error Codes" />
+    <status value="active" />
+    <date value="2022-09-30" />
```

Comment:
Date should be 2023-06-01 (although we also know this will need to be updated to a new date in future anyway)